port_forwarding.sh

```diff
@@ -118,6 +129,11 @@ port="$(echo "$payload" | base64 -d | jq -r '.port')"
 # 2 months is not enough for your setup, please open a ticket.
 expires_at="$(echo "$payload" | base64 -d | jq -r '.expires_at')"
 
+# store port
+echo $port > /opt/piavpn-manual/port || exit 1
```

Comment:
maybe call it forwarded_port instead?

Reply:
Yes why not :)

Reply:
or port_forwarding to refer to the functionality name